PVC/Mesh_voxelisation/README.txt

```diff
@@ -0,0 +1,100 @@
+Polygon mesh voxelisation
+=========================
+
+Adam H. Aitkenhead

```

Comment:
@gmangeat did you check if we have the permission to add in our repos with MIT license?